### DISCUSSION TITLE
Force curve for LJ interaction

### DISCUSSION THREAD

**Discussion by sitangshugk95** [ContactEngineering/Adhesion#89] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/108830413/216178769-6260d7d2-d6f3-4391-9a9f-e906d2798b88.png)

@sannant Doesn't the [force curve](https://github.com/ContactEngineering/Adhesion/blob/master/examples/adhesive_plastic_simulation_cone.ipynb) using the LJ82 interaction look opposite of what it should look like, i.e. repulsive (positive) for shorter distances and attractive (negative) for larger distances? 

![image](https://user-images.githubusercontent.com/108830413/216180146-f2a2cf7e-ded0-4021-9748-26cadd4a323d.png)

Or does the code (somewhere) take the negative of the force again?

**Comment by sannant** [comment, marked as answer]
The label should be gradient, then it is clear. 

the force in direction r is the negative of the gradient of the energy in direction r. 

But the naming and sign conventions and so on in the code are a bit confusing.

"Force" without precision of direction and on which body it acts is always a bit ambiguous.
